numpy/add_newdocs.py

```diff
@@ -463,6 +463,46 @@ def luf(lamdaexpr, *args, **kwargs):
 
     """))
 
+add_newdoc('numpy.core', 'nested_iters',
+    """
+    Create a ordered sequence of iterators. They are nested so that iterating
+    one will reset all the iterators after it in the sequence.
+
+    Parameters 
+    ----------
+    op : ndarray
+        The array to iterate over.
+
+    axes :  tuple of tuples
+        Each item is used as an "op_axes" argument to an nditer
+
+    flags, op_flags, op_dtypes, order, casting, buffersize (optional)
+        See `nditer` parameters of the same name
+
+    Returns
+    -------
+    iters : tuple of nditer
+        An nditer for each item in `axes`
+
+    See Also
+    --------
+    nditer
+
+    Examples
+    --------
+
+    Basic usage
+
+    >>> a = np.arange(12).reshape(2, 3, 2)
+    >>> i, j = np.nested_iters(a, [[0], [1, 2]])
```

Comment:
If we're calling it a tuple above, we should do the same here

Reply:
Would it make sense to swap the second list, so that the result is less trivial?

Reply:
Sorry, that doesn't actually change anything (of course): So maybe `[[2], [0, 1]]`?

Reply:
@mhvk I'm not sure a less trivial result would be good for first-time users who are trying to understand what is going on. This is, after all, a docstring

Reply:
I mostly thought it would be good to show something that changes the order -- for me, when I first looked at your example, I did not understand what had actually happened. But it is not a big deal.